src/renderer/components/Lofi/Cover/Controls/index.tsx

```diff
@@ -39,31 +39,58 @@ class Controls extends React.Component<any, any> {
     this.props.parent.togglePlayPause();
   }
 
-  async forward() {
+  async forward(e: React.MouseEvent) {
     if (this.props.parent.state.spotifyError) {
       return;
     }
 
-    SpotifyApiInstance.fetch('/me/player/next', {
-      method: 'POST',
-    }).then(() => {
-      // Spotify API doesn't update fast enough sometimes, so give it some leeway
-      setTimeout(this.props.parent.listeningTo.bind(this), 2000);
-    });
+    if (e.ctrlKey) {
+      SpotifyApiInstance.fetch(`/me/player`, {
+        method: 'GET',
+      }).then((body) => {
+        const currentProgress = Number(body.progress_ms);
+        SpotifyApiInstance.fetch(`/me/player/seek?position_ms=${currentProgress + 15000}`, {
+          method: 'PUT',
+        }).then(() => {
+          // Spotify API doesn't update fast enough sometimes, so give it some leeway
+          setTimeout(this.props.parent.listeningTo.bind(this), 2000);
+        });
+      });
+    } else {
+      SpotifyApiInstance.fetch('/me/player/next', {
+        method: 'POST',
+      }).then(() => {
+        // Spotify API doesn't update fast enough sometimes, so give it some leeway
+        setTimeout(this.props.parent.listeningTo.bind(this), 2000);
+      });
+    }
```

Comment:
This is old code, I prefer async/await over promise chaining

Also, anything under the if here and for previous playback could be encapsulated into a single method that takes a boolean parameter
Something like

```suggestion
   const jumpSkip = async (isForward: boolean) => {
        const currentlyPlaying = await SpotifyApiInstance.fetch(`/me/player`, {
        method: 'GET',
      })
     const currentProgress = Number(currentlyPlaying.progress_ms);
     const newProgress = isForward ? currentProgress + 15000 : currentProgress - 15000
     await SpotifyApiInstance.fetch(`/me/player/seek?position_ms=${newProgress}`, {
          method: 'PUT',
        });
   }
   
    async forward({ctrlKey}: React.MouseEvent) {
    if (this.props.parent.state.spotifyError) {
      return;
    }
   
    if (ctrlKey) {
      skipDelay(true)
    } else {
      await SpotifyApiInstance.fetch('/me/player/next', {
        method: 'POST',
      })
    }

    // Spotify API doesn't update fast enough sometimes, so give it some leeway
    setTimeout(this.props.parent.listeningTo.bind(this), 2000);
```